src/examples/rover_steering_control/main.cpp

```diff
@@ -178,7 +179,11 @@ void control_attitude(const struct vehicle_attitude_setpoint_s *att_sp, const st
 	/*
 	 * Calculate roll error and apply P gain
```

Comment:
Euler from DCM is cheapest.
 
  Euler(const Quaternion<Type> &q) :
        Vector<Type, 3>()
    {
        *this = Euler(Dcm<Type>(q));
    }